blaze/expr/expressions.py

```diff
@@ -949,15 +955,60 @@ def ndim(expr):
     return len(shape(expr))
 
 
+def drop_field(expr, field, *fields):
+    """Drop a field or fields from a tabular expression.
+
+    Parameters
+    ----------
+    expr : Expr
+        A tabular expression to drop columns from.
+    *fields
```

Comment:
This is intentional, this pattern is to make a *args which requires at least one argument.